pkg/controllers/dynamicbindcontroller/dynamicbindcontroller.go

```diff
@@ -161,11 +162,17 @@ func (d *DynamicBindController) reconcileDynamicBindRequest(dynamicBindRequest *
 	var chosenDynamicBindConfiguration *hfv1.DynamicBindConfiguration
 	var chosenEnvironment *hfv1.Environment
 	var dbcChosen bool
+	var provision bool
 	dbcChosen = false
-
+	provision = true
 	for _, dynamicBindConfiguration := range dynamicBindConfigurations.Items {
 		environment, err := d.hfClientSet.HobbyfarmV1().Environments().Get(dynamicBindConfiguration.Spec.Environment, metav1.GetOptions{})
 
+		if provisionMethod, ok := environment.Annotations["provisioner"]; ok {
```

Comment:
Should we use a more "hobbyfarm-esque" annotation name here? i.e. something like `hobbyfarm.io/provisioner`

Reply:
done.